app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table} (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table}"
+        )
+        return [
+            Actor(*row) for row in actor_cursor
+        ]
+
+    def update(
+            self,
+            id_to_update: int,
+            new_first_name: str,
+            new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table} "
+            "SET first_name = ?, last_name = ? "
+            "WHERE id = ?",
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table} WHERE id = ? ",
+            (id_to_delete,)
+        )
```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.